pkg/config/setup/system_probe_cws.go

```diff
@@ -131,4 +131,5 @@ func initCWSSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	// CWS enforcement capabilities
 	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.enabled", true)
 	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.raw_syscall.enabled", false)
+	cfg.BindEnvAndSetDefault("runtime_security_config.enforcement.rule_source_allowed", []string{"remote-config"})
```

Comment:
So we do not allow it on files ? This seems surprising

Reply:
Yes that is the plan for now

Reply:
I think we should also allow local policies. It seems strange to not allow a local administrator to use the feature when we can use it over remove config

Reply:
I added the `file` source in the default